src/index.ts

```diff
@@ -36,6 +36,29 @@ const listen =
       type: "LISTEN",
     });
 
+interface Grammar {
+    entities: {
+      [index: string]: string;
+        };
+      }
+
+const grammar = {
+  entities: {
+      colour: "pink",
+      object: "lamp",
+      place: "shelf",
+        },
+      };
+
+const modify = (word: string, sentence: string) => {
```

Comment:
Perhaps, not the best name for this function, because there is an impression that it does (modifies) something. It checks if the sentence (utterance) includes the word. Naming it `sentenceIncludesWord` would make it more readable.